src/mame/mame.lst

```diff
@@ -9994,8 +9994,8 @@ cocoh                           // Color Computer with HD6309 CPU
 coco2h                          // Color Computer 2 with HD6309 CPU
 coco2bh                         // Color Computer 2B (uses M6847T1 video chip) with HD6309 CPU
 cocoeh                          // Color Computer (Extended BASIC 1.0) with HD6309 CPU
-cp400                           // Prológica CP400
-cp400c2                         // Prológica CP400 Color II
+cp400                           // Prol�gica CP400
+cp400c2                         // Prol�gica CP400 Color II
```

Comment:
Fix your editor if you screw up text encoding, it's a lot of wasted time fixing it.